README.md

```diff
@@ -101,9 +101,11 @@ There's two optional parameters you can add to this file:
 algolia:
     catch_log_exceptions: true
     index_name_prefix: staging
+    connection_timeout: 2
```

Comment:
I think the default value is 1 second in the algoliasearch-client-php library. We should probably leave at one, or even default it to null.